src/parser/pegen.ts

```diff
@@ -1502,6 +1521,23 @@ export function seq_flatten(p: Parser, seqs: AST[][]): AST[] {
 //     return _Py_Name(uni, Load, EXTRA_EXPR(first_name, second_name));
 // }
 
+class UnreachableException extends Error {}
+
+export function seq_count_dots(seq: TokenInfo[]) {
```

Comment:
I don' t know why but this seems quite a bizarre function - you know you've spent to long on a parser when you're counting dots.

Reply:
lol yeah